gprofiler/metadata/application_identifiers_java.py

```diff
@@ -73,4 +74,11 @@ def get_app_id(self, process: Process) -> Optional[str]:
         )
         if self._APP_NAME_KEY in props:
             return f"spark: {props[self._APP_NAME_KEY]}"
+        args = process.cmdline()
+        try:
+            for idx, x in enumerate(args):
+                if x == self._APP_ID_KEY:
+                    return f"spark: {args[idx+1]}"
+        except Exception:
```

Comment:
Do you expect any particular exception type here? `ValueError` for `idx+1`? Please change to `ValueError` then (it's nicer compared to generic `except Exception` which might catch unexpected things)